frame/support/procedural/src/construct_runtime/expand/call.rs

```diff
@@ -62,6 +62,41 @@ pub fn expand_outer_dispatch(
 		pub enum Call {
 			#variant_defs
 		}
+		#[cfg(test)]
+		impl Call {
+			/// Return a list of the module names together with their size in memory.
+			pub fn sizes() -> &'static [( &'static str, usize )] {
```

Comment:
Can be `const`?

Reply:
Reason why I favour `const` is because checks that can be made `const` means it can be run during compilation time, rather than runtime, and it cuts down the code-compile-execute cycle considerably when testing/debugging.